static/styles/zulip.css

```diff
@@ -548,7 +548,7 @@ p.n-margin {
 .column-middle,
 #compose-content {
     margin-right: $right_sidebar_width;
-    margin-left: $left_sidebar_width;
+    margin-left: 280px;
```

Comment:
I think it makes sense to do this as a function of `$left_sidebar_width`:
```
diff --git a/static/styles/zulip.css b/static/styles/zulip.css
index 589d4827e2..8490d9170e 100644
--- a/static/styles/zulip.css
+++ b/static/styles/zulip.css
@@ -18,7 +18,7 @@ go beneath the header.
 */
 $sidebar_top: calc($header_height + $header_padding_bottom);
 
-/* These need to agree with scroll_bar.js */
+$left_sidebar_collapse_widget_gutter: 10px;
 $left_sidebar_width: 270px;
 $right_sidebar_width: 250px;
 
@@ -532,7 +532,7 @@ p.n-margin {
 


     .column-left .left-sidebar {
         width: $left_sidebar_width;
-        padding-left: 10px;
+        padding-left: $left_sidebar_collapse_widget_gutter;
     }
 
     .column-right .right-sidebar {
@@ -548,7 +548,7 @@ p.n-margin {
 .column-middle,
 #compose-content {
     margin-right: $right_sidebar_width;
-    margin-left: 280px;
+    margin-left: $left_sidebar_width + $left_sidebar_collapse_widget_gutter;
     position: relative;
 }
 
```

I also removed a stale comment from before b0dd063bb8d229b1339588ee20fcef56416454ad.